web/services/hooks/organization_layout.ts

```diff
@@ -1,25 +1,38 @@
 import { useQuery } from "@tanstack/react-query";
-import { OrganizationLayout } from "../lib/organization_layout/organization_layout";
 import { Result } from "../../lib/result";
+import { useJawnClient } from "../../lib/clients/jawnHook";
+import { OrganizationLayout } from "../lib/organization_layout/organization_layout";
 
 const useOrganizationLayout = (
   orgId: string,
   layoutPage: "dashboard" | "requests",
-  initialData?: Result<OrganizationLayout, string>
+  initialData?: Result<OrganizationLayout | null, string>
 ) => {
+  const jawn = useJawnClient();
```

Comment:
Let's replace this with 
`getJawnClient(orgId)`

So we have less hooks on the context